src/services/users.js

```diff
@@ -2,4 +2,8 @@ import { rest } from './rest';
 
 export const login = (data = {}) => rest.post('public/login', data);
 
-export const checkUser = () => rest.get('public/checkUser');
\ No newline at end of file
+export const checkUser = () => rest.get('public/checkUser');
+
+export const createNewUser = ({ firstName, lastName, email, password }) => rest.post('public/user', { firstName, lastName, email, password });
```

Comment:
should be like `export const createUser = user => rest.post('public/user', user);`